ts/Iterable.ts

```diff
@@ -209,11 +209,11 @@ class Iterable {
    * 
    * Note: specify a user by calling Iterable.setEmail or Iterable.setUserId, but NOT both.
    * 
-   * @param {string | undefined} email email address to associate with the current user
+   * @param {string | null} email email address to associate with the current user
    * @param {string | undefined} authToken valid, pre-fecthed JWT the SDK can use to authenticate API requests, optional - if null/undefined, no JWT related action will be taken
    */ 
 
-  static setEmail(email: string | undefined, authToken?: string | undefined) {
+  static setEmail(email: string | null, authToken?: string | null) {
```

Comment:
I think @pauljung14 suggested to have undefined AND null both. 
so it will be `string | undefined | null`

Reply:
You can find my suggestion in the comments of the JIRA ticket

Reply:
Ah gotcha. I was thinking that we wanted the email to be required. But, I will make the change.

Reply:
I think we need to verify what behavior occurs when the user is undefined. Are we logging out the user?